ArduPlane/Plane.h

```diff
@@ -1022,7 +1065,7 @@ class Plane : public AP_HAL::HAL::Callbacks {
     void do_digicam_control(const AP_Mission::Mission_Command& cmd);
     bool start_command_callback(const AP_Mission::Mission_Command &cmd);
     bool verify_command_callback(const AP_Mission::Mission_Command& cmd);
-    void notify_flight_mode(enum FlightMode mode);
+    void notify_mode(const Mode *mode);
```

Comment:
The argument here should never be null; pass a reference instead.

Reply:
fixed